### PR TITLE
Move car

### PR DESCRIPTION
**Pull request by bryanfong-dev-old**

### CODE REVIEW

**Review by geofflin — APPROVED** [review]
Approved.

**Review by jaeroong — APPROVED** [review]
looks good

File: src/server/controllers/userController.js [hunk 1]
```diff
@@ -27,25 +27,22 @@ userController.verifyUser = (req, res, next) => {
...
 
   });
 };
 
 userController.getChallenge = (req, res, next) => {
   client.query('SELECT * FROM codechallenges', (err, results) => {
-    res.set('Content-Type', 'text/plain');
-    res.send(results.rows[0].challenge);
+    res.set('Content-Type', 'application/json');
+    const i = Math.floor(Math.random() * results.rows.length);
```
Comment (Donaldblanc) [inline src/server/controllers/userController.js:44]: smart! I like this !

File: src/server/controllers/userController.js [hunk 2]
```diff
@@ -27,25 +27,22 @@ userController.verifyUser = (req, res, next) => {
...
+    res.send(JSON.stringify(results.rows[i].challenge));
```
Comment (Donaldblanc) [inline src/server/controllers/userController.js:45]: should add error handling but for now this looks great!

**Review by Donaldblanc — APPROVED** [review]

**Pull request merged**